### PR TITLE
Add sym/cmp for nRF52 series

### PR DESCRIPTION
**Pull request by dbrgn**
- [x] nRF52810: https://infocenter.nordicsemi.com/pdf/nRF52810_PS_v1.3.pdf (Page 387)
- [x] nRF52832: https://infocenter.nordicsemi.com/pdf/nRF52832_PS_v1.4.pdf (Page 13)
- [x] nRF52840: https://infocenter.nordicsemi.com/pdf/nRF52840_PS_v1.1.pdf (Page 575)

### CODE REVIEW

**Comment by dbrgn** [comment]
For now, I added three common MCUs from the nRF52 series. The nRF52811 and nRF52833 are still missing, but could be added later on.

The nRF52840 has quite a big symbol that could be split up into 3 symbols (General/P0/P1), but I'll do that in a separate PR once the unified symbol(s) have been reviewed.

I linked to the datasheets in the PR description. The pinout list is on the page specified next to the link.

Ready to review! Maybe @rnestler would also like to take a look?

**Comment by rnestler** [comment]
Can you copy paste the symbol images to the top comment? Gives a nice overview about what to expect :slightly_smiling_face:

File: cmp/0a91342f-3d61-45c0-ae68-4be3073542d5/component.lp [hunk 1]
```diff
@@ -0,0 +1,138 @@
+(librepcb_component 0a91342f-3d61-45c0-ae68-4be3073542d5
+ (name "nRF52810-QCxx")
+ (description "Bluetooth 5.2 SoC supporting Bluetooth Low Energy.\n\nQFN32 package variant.")
```
Comment (rnestler) [inline cmp/0a91342f-3d61-45c0-ae68-4be3073542d5/component.lp:3]: Why is the component package specific? Because of the number of pins?

Comment (dbrgn) [reply cmp/0a91342f-3d61-45c0-ae68-4be3073542d5/component.lp:3]: Yep, and sometimes different pins being available (or not). I would have merged the symbols if it was possible.

**Comment by dbrgn** [comment]
> Can you copy paste the symbol images to the top comment?

There are 7 different symbols, I'd have to rescale the images first (since the ones generated by LibrePCB are quite huge) and it wouldn't really help much for reviewing anyways :slightly_smiling_face: Or would you prefer to review screenshots instead of the actual symbols?

(Btw, what I do for reviewing is moving out all pins by 1 step. Then for every checked pin in the datasheet, I move the corresponding symbol pin back to its position. This way, I can see if a pin in the symbol is missing in the datasheet, and vice versa.)

**Review by ubruhin — APPROVED** [review]
I only quickly reviewed it in terms of library guidelines etc., but I didn't compare it with the datasheets (too many pins :grin:) So far it looks good, the only small inconsistency I found is that some of the symbols are added to only one category, and some are added to two categories.

**Comment by ubruhin** [comment]
Ah and I forgot: Could you rebase the branch before merging? Just to get CI run on this branch...

**Comment by dbrgn** [comment]
> the only small inconsistency I found is that some of the symbols are added to only one category, and some are added to two categories

Ah, I'll fix that. This is due to https://github.com/LibrePCB/LibrePCB/issues/696.

**Comment by dbrgn** [comment]
@ubruhin fixed, merged & version bumped. can you add the library to the API?

**Comment by ubruhin** [comment]
> can you add the library to the API?

Done :rocket:

**Pull request merged**